### PR TITLE
Override default settings

### PR DESCRIPTION
**Pull request by adriendupuis**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | 
| Versions      | master, 4.4
| Edition       | 

It's best practice to override default settings from own config files than to edit a vendor/ file that could be overwritten during a dependencies update.

Inspired by #1660

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/customer_management/cp_configuration.md [hunk 1]
```diff
@@ -29,16 +29,17 @@ ibexa:
 ## Roles and Policies
 
 You can add custom roles to your installation
-by adding them to the configuration file `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`:
+by adding them to `ibexa.site_access.config.default.corporate_accounts.roles`
```
Comment (DominikaK) [inline docs/customer_management/cp_configuration.md:32]:
```suggestion
by adding them under the `ibexa.site_access.config.default.corporate_accounts.roles` key
```

File: docs/customer_management/cp_configuration.md [hunk 2]
```diff
@@ -29,16 +29,17 @@ ibexa:
...
 
 ```yaml
 parameters:
     ibexa.site_access.config.default.corporate_accounts.roles:
-        admin: Company Admin
-        buyer: Company Buyer
         custom_role: Company Assistant
 ```
 
+You can do it per Site Access or Site Access grup by using [SiteAccess-aware configuration](multisite/siteaccess_aware_configuration.md).
```
Comment (DominikaK) [inline docs/customer_management/cp_configuration.md:41]:
```suggestion
You can do it per SiteAccess or SiteAccess group by using [SiteAccess-aware configuration](siteaccess_aware_configuration.md).
```

File: docs/customer_management/cp_configuration.md [hunk 3]
```diff
@@ -29,16 +29,17 @@ ibexa:
 ## Roles and Policies
 
 You can add custom roles to your installation
-by adding them to the configuration file `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`:
+by adding them to `ibexa.site_access.config.default.corporate_accounts.roles`
+(to the default already set in `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`):
 
 ```yaml
 parameters:
```
Comment (DominikaK) [inline docs/customer_management/cp_configuration.md:36]: Could we actually have a full example of the override here?

**Review by DominikaK — CHANGES_REQUESTED** [review]

File: docs/customer_management/cp_configuration.md (hunk 2 above)
Comment (adriendupuis) [reply docs/customer_management/cp_configuration.md:41]: "grup" is so cute…

File: docs/customer_management/cp_configuration.md (hunk 3 above)
Comment (adriendupuis) [reply docs/customer_management/cp_configuration.md:36]: @DominikaK I'm not sure about what you're asking for.
I'll make test at lest to validate that the custom role below is added to the existing ones and replacing the whole.

Comment (DominikaK) [reply docs/customer_management/cp_configuration.md:36]: Sorry, disregard that comment, I misunderstood.

**Review by DominikaK — APPROVED** [review]

File: docs/customer_management/cp_configuration.md [hunk 4]
```diff
@@ -29,7 +29,8 @@ ibexa:
 ## Roles and Policies
 
 You can add custom roles to your installation
-by adding them to the configuration file `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`:
+by listing them under the `ibexa.site_access.config.default.corporate_accounts.roles` key
+(it overwrites the default list set in `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml`, the following example keep them by redeclaring them):
```
Comment (adriendupuis) [inline docs/customer_management/cp_configuration.md:33]: @DominikaK Test done, it doesn't add, it replaces. I'm not satisfied with my wording. Can you help me, please?

Comment (DominikaK) [inline docs/customer_management/cp_configuration.md:33]:
```suggestion
by listing them under the `ibexa.site_access.config.default.corporate_accounts.roles` key.
This key overwrites the default list set in `vendor/ibexa/corporate-account/src/bundle/Resources/config/default_settings.yaml` (the following example redeclares them for clarity):
```

**Review by DominikaK — APPROVED** [review]

File: docs/customer_management/cp_configuration.md (hunk 4 above)
Comment (adriendupuis) [reply docs/customer_management/cp_configuration.md]: Done in 0e63874a3c9db3e891de85e1ec877437d62bef18

**Review by reithor — APPROVED** [review]

File: docs/customer_management/cp_configuration.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (DominikaK) [reply docs/customer_management/cp_configuration.md]: Good catch!

**Pull request merged**